### DISCUSSION TITLE
Please consider introducing MMR to Turbo game

### DISCUSSION THREAD

**Discussion by cosmic-void-man** [ValveSoftware/Dota2-Gameplay#2177] [Ideas]
To the best of my understanding, a significant portion of the Dota2 players enjoy Turbo games, and a great number of them only play Turbo games. As a game mode that is almost as popular as All Pick games, Turbo currently does not have a Ranked version of it, on the contrary. With the time going on, it can be expected that the players who only remain in Turbo games will become more experienced and competitive. Thus, it should be a good idea to introduce Ranked Turbo games as well as display the MMR and medal for such competitive Turbo games. The implementation of this idea is supposed to be straightforward, because it is quite similar to what have been done in Ranked All Pick games. Thank you for your consideration.

**Comment by jameswilliam20** [comment]
Wow, amazing post thank you. if you want more tech information click here.
[does aldi take apple pay](https://www.takeapplepayzone.com/)
[ludo earning money app](https://www.alltechdownload.com/)

**Comment by Kazim4321** [comment]
The gaming industry is very popular these days and almost more than 50% of youngsters are playing games and making their youtube channels and live streaming their game play. I suggest to all those who are related to this field must try to get some [link building services](https://rankqu.com/link-building-services/) for their channels, this will definitely help to rank their channels.

**Comment by MaryNegron** [comment]
Nowadays, nearly half of young people play video games, create YouTube channels, and live stream their gameplay. The gaming sector is extremely popular [skintes](https://skintes.com/collections/image-skincare). I advise everyone involved in this industry to try to hire some link building services for their channels, as this will undoubtedly help them rank.

**Comment by onesmartsheep1** [comment]
Link Building, for sure, can help get a site more popular. it further helps to get the appreciation a platform really deserves. But, it is quite a tricky taste. I myself has experienced it[.](https://onesmartsheep.com/) I hope we soon can find the resources and pathologies using which we will be able to builds a site reputation.

**Comment by avaannu** [comment]
Here is a site from where you can download free [gaming](https://modapksquad.com/) apps.

**Comment by avaannu** [comment]
You have to play [anime games](https://modapksquad.com/) to enjoy yourself.
